lib/user_interface.rb

```diff
@@ -0,0 +1,108 @@
+require_relative 'menu.rb'
+require_relative 'order.rb'
+require 'twilio-ruby' # put your own credentials here
+require_relative 'SMS.rb'
+
+class UserInterface
+  def initialize
```

Comment:
However you have a LOT of functionality that we didn't ask for. If the client then turned around and said they were going to plug this software into an ipad app, they would not be happy to pay you for the time you spent building this complex UI.